tests/acceptance/features/apiAsyncUpload/delayPostprocessing.feature

```diff
@@ -11,22 +11,22 @@ Feature: delay post-processing of uploaded files
   @issue-5326
   Scenario Outline: user sends GET request to the file while it's still being processed
     Given user "Alice" has uploaded file with content "uploaded content" to "/file.txt"
-    When user "Alice" requests "<dav_path>" with "GET" without retrying
+    When user "Alice" requests "<dav-path-version>" with "GET" without retrying
```

Comment:
Let it be dav_path, it's different. `dav_path` is used for the path of endpoints, whereas `dav-path`, `dav-path-version`, and `dav_version` are used for the WebDAV versions.